PhysicsTools/PatAlgos/python/producersLayer1/jetProducer_cfi.py

```diff
@@ -47,7 +47,12 @@
         cms.InputTag("pfCombinedInclusiveSecondaryVertexV2BJetTags"),
         cms.InputTag("softPFMuonBJetTags"),
         cms.InputTag("softPFElectronBJetTags"),
-        cms.InputTag("pfCombinedMVABJetTags")
+        cms.InputTag("pfCombinedMVABJetTags"),
+        cms.InputTag("pfCombinedSecondaryVertexSoftLeptonBJetTags"),
```

Comment:
I guess this is something that was present in my previous build but then got removed and I mistakenly replaced. Fixing it.